Sources/SparkConnect/DataFrame.swift

```diff
@@ -217,7 +217,7 @@ public actor DataFrame: Sendable {
             values.append(str.asString(i))
```

Comment:
This is still stored the value as string? So we still cannot insert column values into Row in their types?

Reply:
Yes~  This is a part-one implementation to introduce `Row` struct like the note in the PR description.
> Note that Row is added to support general type fields, but this PR replaces the existing API's [String?] signature into Row-based signature. The detailed one-to-one mapping among other types will be handled later.

Reply:
Got it. Thank you.